activitysim/cli/run.py

```diff
@@ -283,7 +285,7 @@ def run(args):
         # Memory sidecar is only useful for single process runs
         # multiprocess runs log memory usage without blocking in the controlling process.
         mem_prof_log = state.get_log_file_path("memory_profile.csv")
-        from ..core.memory_sidecar import MemorySidecar
+        from activitysim.core.memory_sidecar import MemorySidecar
```

Comment:
The relative import here caused problems for my runs - suggest changing it to absolute.